x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/schema.ts

```diff
@@ -42,4 +55,31 @@ export const pipelineFormSchema: FormSchema<PipelineForm> = {
     }),
     formatters: [toInt],
   },
+  _meta: {
+    label: i18n.translate('xpack.ingestPipelines.form.metaFieldLabel', {
+      defaultMessage: '_meta field data (optional)',
+    }),
+    helpText: i18n.translate('xpack.ingestPipelines.form.metaHelpText', {
+      defaultMessage: 'Use JSON format.',
```

Comment:
nit: should we use an example here also like we do in index management create template UI?

Reply:
Yes, indeed an example seems to be helpful so I added it in the next commit, but had to rename this file to `schema.tsx` so that it can contain React components.